pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/pendingack/impl/PendingAckHandleImpl.java

```diff
@@ -946,6 +946,7 @@ public void completeHandleFuture() {
 
     public void exceptionHandleFuture(Throwable t) {
```

Comment:
By the way, we should correct the log print to make users can know what is wrong.